doc_k8s/linux/docker/Dockerfile.script

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
```

Comment:
I think the extension .script is not really obvious and does allow to suppose this is a bash/sh script.
Could you please adapt the extension for .sh?